conans/test/unittests/util/detect_libc_test.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+
+from parameterized import parameterized
+
+from conan.internal.api.detect_api import _parse_gnu_libc, _parse_musl_libc
+
+
+class DetectedLibcTest(unittest.TestCase):
```

Comment:
We are trying not to use ``unittest`` anymore, but only ``pytest``. It is ok, no need to change it, just FYI for future.

Reply:
Oh, I don't mind. I've updated that.